src/compat/posix/fnmatch/fnmatch.c

```diff
@@ -0,0 +1,160 @@
+/**
+ * @file
+ * @brief fnmatch_function
+ *
+ * @date 20.12.18
+ * @author Filipp Chubukov
+ */
+
+#include <stdlib.h>
+#include <errno.h>
+#include <fnmatch.h>
+
+static int bracket_check(const char *pattern) {
+	while (*pattern != ']') {
+		if (*pattern == '/') {
+			return 0;
+		}
+		pattern++;
+	}
+	return 1;
+}
+
+static int fnmatch_string(const char *pattern, const char *string, int flags) {
+	int flag = 0;
+	int denial_flag = 0;
+	while (*pattern != '\0') {
+		flag = 0;
```

Comment:
These two variables are used only in one `case`, so it's better to move these two lines there.

Reply:
moved